modules/creating-a-data-science-project.adoc

```diff
@@ -4,7 +4,7 @@
 = Creating a data science project
 
 [role='_abstract']
-To start your data science work, create a data science project. Creating a project helps you organize your work in one place. You can also enhance the capabilities of your data science project by adding workbenches, adding storage to your project's cluster, adding data connections, and adding model servers.
+To start your data science work, create a data science project. Creating a project helps you organize your work in one place. You can also enhance the capabilities of your data science project by adding workbenches, adding storage to your project's cluster, adding data connections, adding model servers, and adding bias monitoring to models.
```

Comment:
It might be time to format this list as a bullet list? For example:

```

You can also enhance your data science project by adding the following functionality:

* Workbenches
* Storage for your project's cluster
* Data connections
* Model servers
* Bias monitoring for your models
````